libs/agno/pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
   "tomli",
   "typer",
   "typing-extensions",
+  "opensearch-py"
```

Comment:
We prefer not to add it here, because it increases the size for everyone. So rather remove pls.